CHANGELOG.md

```diff
@@ -7,6 +7,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.7.0] - November, 22, 2022
+
+### Design kit
+
+#### Added
+
+- [Icons] New icon UI/Navigation/swap-user
+- [Labels and Tags] New tag variants with rounded corners
+
+### React Components
+
+#### Added
+
+- [Labels and Tags] New tag variants with rounded corners
+
+#### Changed
+
+- [Input fields] Input fields' buttons have larger clickable area around them
```

Comment:
Should we put the affected components straight in here, like `[DateInput, SearchInput, PasswordInput]` ?

Reply:
Yes